Dockerfile

```diff
@@ -15,12 +15,11 @@
 
 COPY --from=builder /app .
 
+# Install dependencies
 RUN pip install --upgrade pip && \
     pip install -r requirements.txt
 
-RUN python manage.py migrate
-
 EXPOSE 8080
 
-# Run database migrations and start the Django application
-ENTRYPOINT ["python", "manage.py", "runserver", "0.0.0.0:8080"]
+# Run database migrations and start Django application
+ENTRYPOINT ["sh", "-c", "python manage.py migrate && python manage.py runserver 0.0.0.0:8080"]
```

Comment:
Using `sh -c` in the ENTRYPOINT can be problematic if you need to pass additional arguments to the container at runtime. Consider using a shell script or directly executing the Python commands to allow for more flexibility.